velox/connectors/hive/HiveConnector.cpp

```diff
@@ -62,6 +65,44 @@ HiveConnector::HiveConnector(
           std::make_unique<FileHandleGenerator>(std::move(properties))),
       executor_(executor) {}
 
+std::unique_ptr<DataSource> HiveConnector::createDataSource(
+    const RowTypePtr& outputType,
+    const std::shared_ptr<ConnectorTableHandle>& tableHandle,
+    const std::unordered_map<
+        std::string,
+        std::shared_ptr<connector::ColumnHandle>>& columnHandles,
+    ConnectorQueryCtx* connectorQueryCtx) {
+  dwio::common::ReaderOptions options(connectorQueryCtx->memoryPool());
+  options.setMaxCoalesceBytes(
+      HiveConfig::maxCoalescedBytes(connectorQueryCtx->config()));
+  options.setMaxCoalesceDistance(
+      HiveConfig::maxCoalescedDistanceBytes(connectorQueryCtx->config()));
+  return std::make_unique<HiveDataSource>(
+      outputType,
+      tableHandle,
+      columnHandles,
+      &fileHandleFactory_,
+      connectorQueryCtx->expressionEvaluator(),
+      connectorQueryCtx->allocator(),
+      connectorQueryCtx->scanId(),
+      HiveConfig::isFileColumnNamesReadAsLowerCase(connectorQueryCtx->config()),
+      executor_,
+      options);
+}
+
+std::unique_ptr<DataSink> HiveConnector::createDataSink(
```

Comment:
Same here.

Reply:
This is just being moved from the `HiveConnector.h` header to the cpp file.